quantum/nodes/not-gate/not-gate.html

```diff
@@ -0,0 +1,35 @@
+<script type="text/javascript">
+  RED.nodes.registerType("not-gate", {
+    category: "quantum",
+    color: "#FA4D56",
+    defaults: {
+      name: { value: "" },
+    },
+    inputs: 1,
+    outputs: 1,
+    //icon: "function.svg",
+    paletteLabel: "not",
+    icon: "font-awesome/fa-close",
+    align: 'right',
+    label: function () {
+      return this.name || "not";
+    },
+    inputLabels: function(index) {
+      return "qubit " + index;
+    },
```

Comment:
I would change `inputLabels` & `outputLabels` to be:

`inputLabels = "Qubit"`
`outputLabels = "Qubit"`

Reply:
After that, I am happy to merge!